x/mongo/driver/topology/connection.go

```diff
@@ -292,10 +294,16 @@ func (c *connection) readWireMessage(ctx context.Context, dst []byte) ([]byte, e
 	if err != nil {
 		// We closeConnection the connection because we don't know if there are other bytes left to read.
 		c.close()
+		var message string
```

Comment:
Github doesn't let me comment in the right place because the code didn't change, but we do another `io.ReadFull` call below to read the rest of the message after we get the header. Do we need to change the message in that codepath too?

Reply:
done